circle.yml

```diff
@@ -1,51 +0,0 @@
-dependencies:
-  override:
-    # chmod to address a Circle "bug", since gcloud lives at /opt
-    # and wants to make a .staging directory (as a sibling!)
-    # when upgrading or installing new components.
-    - sudo chmod o+w /opt
-    # CircleCI's version of pip can be out of date; make sure it's up-to-date first.
-    - pip install --upgrade pip
-    # Then install requirements needed to run our tools.
-    - pip install --upgrade httplib2 jira oauth2client parallel pylint
-    - pip install -I alembic==0.9.1
-
-machine:
-  services:
-    - mysql
-
-test:
-  override:
-    - ./ci/test.sh
-
-deployment:
-  # Deploy master to the pmi-drc-api-test project whenever the tests pass
-  test:
-    branch: master
-    commands:
-      - ./ci/activate_creds.sh ~/gcloud-credentials.key
-      - ./tools/upgrade_database.sh -i https://pmi-drc-api-test.appspot.com --creds_file ~/gcloud-credentials.key:
-          pwd:
-            rest-api
-      - ./ci/deploy.sh pmi-drc-api-test config/config_test.json ~/gcloud-credentials.key
```

Comment:
It appears that `deploy.sh` didn't make the cut here (both here and staging); definitely need that to run in the new config.

Reply:
Yeah, it looks like they want it configured in a completely different way. From the inline comments in the new file:

```
    # Deployment
    # Your existing circle.yml file contains deployment steps.
    # The config translation tool does not support translating deployment steps
    # since deployment in CircleCI 2.0 are better handled through workflows.
    # See the documentation for more information https://circleci.com/docs/2.0/workflows/
```